From 32e97018435385dd278e44f83ad5e689137cda17 Mon Sep 17 00:00:00 2001
From: kalebmckale <kalebmckale@users.noreply.github.com>
Date: Sat, 19 Aug 2023 12:13:53 -0400
Subject: [PATCH] Fix typo in _pkcs11.pyx

Changed typo on L380 from `CK_OBJECT_HANDLE` to `CK_SESSION_HANDLE`.

(cherry picked from commit f5371472376404936fc05a187cc501567af73871)
---
 pkcs11/_pkcs11.pyx | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkcs11/_pkcs11.pyx b/pkcs11/_pkcs11.pyx
index 7f9deef..7022ce6 100644
--- a/pkcs11/_pkcs11.pyx
+++ b/pkcs11/_pkcs11.pyx
@@ -356,7 +356,7 @@ class Session(types.Session):
     """Extend Session with implementation."""
 
     def close(self):
-        cdef CK_OBJECT_HANDLE handle = self._handle
+        cdef CK_SESSION_HANDLE handle = self._handle
 
         if self.user_type != UserType.NOBODY:
             with nogil:
-- 
2.42.0

